edxval/tests/constants.py

```diff
@@ -30,13 +30,33 @@
 """
 ENCODED_VIDEO_DICT_MOBILE = dict(
     url="http://www.meowmix.com",
-    file_size=4545,
-    bitrate=6767,
+    file_size=11,
+    bitrate=22,
 )
 ENCODED_VIDEO_DICT_DESKTOP = dict(
     url="http://www.meowmagic.com",
-    file_size=1212,
-    bitrate=2323,
+    file_size=33,
+    bitrate=44,
+)
+ENCODED_VIDEO_DICT_MOBILE2 = dict(
+    url="http://www.woof.com",
+    file_size=55,
+    bitrate=66,
+)
+ENCODED_VIDEO_DICT_DESKTOP2 = dict(
+    url="http://www.bark.com",
+    file_size=77,
+    bitrate=88,
+)
+ENCODED_VIDEO_DICT_MOBILE3 = dict(
+    url="http://www.ssss.com",
+    file_size=1111,
+    bitrate=2222,
+)
+ENCODED_VIDEO_DICT_DESKTOP3 = dict(
+    url="http://www.hiss.com",
+    file_size=3333,
+    bitrate=4444,
```

Comment:
Nitpick: you don't necessarily need a  new encoded_video_dict for each encoding.  The same one can be reused, no? Otherwise, it adds extra baggage to the test code.